misc/config_template.in.hjson

```diff
@@ -318,9 +318,13 @@
         // Make sure to adjust 'prettyBoardName' to your liking in your config before enabling
 
         mrc: {
-            enabled             : false
-            serverHostname      : 'mrc.bottomlessabyss.net'
-            serverPort          : 5000
+            enabled         : false
+            serverHostname  : 'mrc.bottomlessabyss.net'
+            serverPort      : 5000
+            serverSslPort   : 5001
+            useSsl          : false
```

Comment:
Is there any reason not to use TLS by default?

Reply:
I usually leave it disabled because in some case it requires an optional library not always installed.
If that is not the case here, please set it as default to true for sure.

Reply:
Awesome, updated. Should be fine since Node has TLS and that's all that's really happening in this case